utils/vim/Makefile

```diff
@@ -96,7 +102,7 @@ endef
 
 define Package/vim-fuller/description
  Vim is an almost compatible version of the UNIX editor Vi.
- (Big build)
+ (Huge build)
 endef
```

Comment:
I know it is not related to your changes, but it would be great if the description explained the difference between Tiny/Normal/Huge builds

Reply:
Thank you!! :) Huge improvement.